vision/api/ProductSearch/ImportProductSets.cs

```diff
@@ -0,0 +1,71 @@
+﻿using CommandLine;
+using Google.Cloud.Vision.V1;
+using Google.LongRunning;
+using System;
+
+namespace GoogleCloudSamples
+{
+    [Verb("import_product_set", HelpText = "Import a product set from GCS")]
+    class ImportProductSetOptions : BaseOptions
+    {
+        [Value(2, HelpText = "Google Cloud Storage URI. Target files must be in Product Search CSV format.")]
```

Comment:
I don't remember what the integer here is used for. Just calling this value out, incase it shouldn't be 2 👍

Reply:
The integer value is the index of the argument passed into the command. BaseOptions provides indexes 0 (project ID) and 1 (region).

Do you think it would be better to use flags instead?